knox-ui/src/Main_components/Home.js

```diff
@@ -1,35 +1,39 @@
-import React from 'react'
+import React, { useState } from 'react';
 import Button from 'react-bootstrap/Button';
 import SearchBar from '../Shared_components/SearchBar';
-import { useState } from 'react';
-import PaginatedSearchResults from '../Shared_components/PaginatedSearchResults'
-import AdvancedSidebar from '../Shared_components/AdvancedSideBar'
+import PaginatedSearchResults from '../Shared_components/PaginatedSearchResults';
+import AdvancedSidebar from '../Shared_components/AdvancedSideBar';
+import GetSearchResult from '../Services/SearchService';
+import SearchOptions from '../Models/SearchOptionsModel';
 import '../Css/HomePage.css';
 
-const Home = (SuggesterConnection) => {
-    const [open, setOpen] = useState(false);
-    const [searchResults, setSearchResults] = useState([])
-    const [searching, setSearching] = useState(false);
-    const [firstSearchMade, setFirstSearchMade] = useState(false)
-    // OBS OBS OBS! if more databases are added, add the names here as well as in the checkboxes on Advanced Sidebar!
-    const [advancedOptions, setAdvancedOptions] = useState(["Grundfos A/S", "Nordjyske Medier"])
-    
+const Home = () => {
+    const useSuggester = false; //Suggester temp dissable
+    const [isOpen, setIsOpen] = useState(false);
+    const [searchResults, setSearchResults] = useState([]);
+    const [isSearching, setIsSearching] = useState(false);
+    const [isFirstSearchMade, setIsFirstSearchMade] = useState(false);
+    const [searchOptions, setSearchOptions] = useState(new SearchOptions());
+
     const onClick = (searchText) => {
-        if (searching === true) return
-        if (searchText === "" || advancedOptions.length===0) {
-            setSearchResults([])
-            return
+        if (isSearching === true) {
+            return;
+        }
+
+        if (searchText !== undefined || searchText !== '') {
```

Comment:
Ved ikke om `.trim()` findes, men det ville være godt at tilføje noget lignende her, hvis vi ikke ønsker at man skal kunne søge med ren whitespace
```suggestion
        if (searchText !== undefined || searchText.trim() !== '') {
```